Cargo.toml

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This should be the apache license header, not the asf header.

Reply:
Is just `// SPDX-License-Identifier: Apache-2.0` sufficient? I notice that is the approach used in the main substrait repo, at least in the proto files.

Reply:
Actually yes. LOL. SPDX replaces the big header. I just had a mental lapse :)